### PR TITLE
Reimplement AT=>DX migration with inplace migrator

### PR DESCRIPTION
**Pull request by jone**
Use ftw.upgrade's new inplace migrator for migrating subsites from AT to
DX. The existing upgrade step is rewritten in order to run only when not
yet migrated.

The migrator is configured to not migrate fields which we had on AT but
no longer on DX. Instead, those values are backed up in the annotations
of the new subsite.

For projects which add behaviors to the subsite for those old fields,
the values can be gotten from the annotations and re-set after applying
the behavior. This is the responsibility of the integration project.

### CODE REVIEW

File: sources.cfg [hunk 1]
```diff
@@ -5,3 +5,11 @@ extensions = mr.developer
 auto-checkout =
     ftw.simplelayout
     ftw.mobile
+
+# https://github.com/4teamwork/ftw.upgrade/pull/111
+    ftw.upgrade
```
Comment (jone) [inline sources.cfg]: 🚧 https://github.com/4teamwork/ftw.upgrade/pull/111

File: sources.cfg [hunk 2]
```diff
@@ -5,3 +5,11 @@ extensions = mr.developer
...
+
+
+[branches]
+# https://github.com/4teamwork/ftw.upgrade/pull/111
+ftw.upgrade = jone-inplace-migration
```
Comment (jone) [inline sources.cfg]: 🚧 https://github.com/4teamwork/ftw.upgrade/pull/111

File: sources.cfg (hunk 1 above)
Comment (mbaechtold) [inline sources.cfg]: ftw.upgrade 2.0.0 has been released.

File: sources.cfg (hunk 2 above)
Comment (mbaechtold) [inline sources.cfg]: ftw.upgrade 2.0.0 has been released

**Pull request merged**